wasmbinding/message_plugin.go

```diff
@@ -985,7 +985,15 @@ func (m *CustomMessenger) addSchedule(ctx sdk.Context, contractAddr sdk.AccAddre
 		})
 	}
 
-	err := m.CronKeeper.AddSchedule(ctx, addSchedule.Name, addSchedule.Period, msgs)
+	var executionStage crontypes.ExecutionStage
+
+	if v, ok := crontypes.ExecutionStage_value[addSchedule.ExecutionStage]; !ok {
+		executionStage = crontypes.ExecutionStage_END_BLOCKER
+	} else {
+		executionStage = crontypes.ExecutionStage(v)
+	}
```

Comment:
do we even need that? why don't enforce stage being set? is that for backwards compatibility? if yes, with what? also, as far as I'm concerned, you can simply make the `ExecutionStage_END_BLOCKER` the first value in the enum (i.e. set it the value 0) and it will be the default value in case of explicit parameter absence

Reply:
Remade